dev/io.openliberty.microprofile.reactive.messaging.internal_fat/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2019, 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```

Comment:
```suggestion
# Copyright (c) 2019, 2023 IBM Corporation and others.
```

Copyright headers should have two dates, the year it was first published and the year it was last changed.

Reply:
@Azquelt given this file is new to this year - it was a mistake on my part originally - it should just be 2023.